test/test_p2a.py

```diff
@@ -127,7 +127,7 @@ class TestP2A(unittest.TestCase):
 
     def setUp(self):
 
-        self.p2a = Phone2Action(app_id='an_id', app_key='app_key')
+        self.p2a = Phone2Action(app_id='an_id', app_key='app_key').capitol_canary
```

Comment:
Ahhh just noticed this. Sorry for assuming we were done!  😭 

@bzupnick, can you leave this as `self.p2a = Phone2Action(app_id='an_id', app_key='app_key')`? We want people to be able to keep using the Phone2Action class for now, with just a warning, which means we need to test that the wrapper is actually working by calling the class's methods directly, rather than testing `.capitol_canary`'s methods.

Reply:
The reason I did that was since below we use things like `self.p2a.client.uri` which doesn't exist in p2a anymore.

But now that I write this out, I wonder if that means that my solution is incomplete and I should _also_ bring in not just the functions, but also the class variables 🤔

Reply:
Let's try overriding `__getattr__`. `__getattr__` is a "magic method" that is only called when the attribute is not found. If we override it to look for the same attribute on CapitolCanary, that should move over all the attributes. 

Should look something like this (this is from memory, the syntax may be off and its definitely oversimplified):

```

class Phone2Action:

  self.__init__()
    self.capitolcanary = CapitolCanary

  self.__getattr__(self, name):
    return getattr(self.capitolcanary, name)
```